### ISSUE TITLE
preprocessor parseing ignored that should be ignored 

### ISSUE THREAD

**Issue by luisgutz** [SystemRDL/systemrdl-compiler#64]
Consider the following RDL file (bug.rdl):

```
reg reg_arr_r {
    field {} x;
};

addrmap vog_preprocessor_bug {
    `ifdef ARRSZ
    reg_arr_r regarr[`ARRSZ];
    `endif
};
```

The compiler will complain with this error:
```
bug.rdl:7:19: fatal: Macro '`ARRSZ' has not been defined
    reg2_t reg2a[`ARRSZ]
```

But the expected behaviour would for the preprocessor to ignore that line, as ARRSZ is not defined.

**Comment by amykyta3** [comment]
Wow can't believe i missed that :laughing: 
Thanks for reporting it! Should be quick to fix.

**Comment by amykyta3** [comment]
Just released `v1.13.1` where this should be fixed.

Thanks again for reporting!
